File: examples/nodejs/cli/find_did.ts
import { get } from './sock.js'

(async () => {
  const json = await get('/identifiers/did:nodex:test:EiD_ZSrS4E4FZruAIJnMt1KjvH1HvwCRYdnIzYpQr4vsuQ')

  console.log(json)
})()

File: examples/python/v2/cli/find_did.py
from sock import get


# PLEASE WRITE destination_did
destination_did = (
    "did:nodex:test:DummyDummyDummyDummyDummyDummyDummyDummyDummyD"
)

endpoint = "/identifiers/" + destination_did
json_response = get(endpoint)

print("The response is as follows.\n")
print(json_response)
